azkaban-common/src/main/java/azkaban/utils/Props.java

```diff
@@ -811,20 +812,28 @@ public void storeFlattened(OutputStream out) throws IOException {
   }
 
   /**
+   * Returns a map of all the flattened properties, the item in the returned map is sorted alphabetically
+   * by the key value.
+   *
+   *
+   * @Return
+   */
+  public Map<String,String> getFlattened(){
+    TreeMap<String,String> returnVal = new TreeMap<String,String>(); 
+    returnVal.putAll(getMapByPrefix(""));
```

Comment:
instead of using getMapByPrefix(), maybe just use the Prop class's _current variable?